var/spack/repos/builtin/packages/py-pyedr/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyedr(PythonPackage):
+    """Pyedr provides a means of reading a Gromacs EDR binary XDR file and return
+    its contents as a dictionary of NumPy arrays"""
+
+    homepage = "https://github.com/MDAnalysis/panedr"
+    pypi = "pyedr/pyedr-0.7.1.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.7.1", sha256="ad7ccdeb739399acd11a25f2d2413ebb46a54223059a2b902ac604d29fabd767")
+
+    depends_on("python@3.6:")
+
+    depends_on("py-numpy@1.19.0:", type=("build", "run"))
```

Comment:
Where do you see the version requirement?

Reply:
It it here: https://github.com/MDAnalysis/panedr/blob/7268391a60e14b7c065eb1f7c074e14c29e59515/pyedr/requirements.txt#L1 However, it did not make it to `setup.cfg`.

Reply:
```suggestion
    # Minimal NumPy version only specified in requirements.txt
    depends_on("py-numpy@1.19.0:", type=("build", "run"))
```